gix/tests/gix/revision/spec/from_bytes/reflog.rs

```diff
@@ -78,10 +75,13 @@ fn by_index() {
 }
 
 #[test]
-fn by_date_is_planned_until_git_date_crate_is_implements_parsing() {
+fn by_date() {
     let repo = repo("complex_graph").unwrap();
-    assert!(matches!(
-        parse_spec_no_baseline("main@{1979-02-26 18:30:00}", &repo).unwrap_err(),
-        Error::Planned { .. }
-    ));
+
+    let spec = parse_spec_no_baseline("main@{1979-02-26 18:30:00}", &repo).unwrap();
+
+    assert_eq!(
```

Comment:
Now there can easily be more tests which are then compared to what Git does - here this can be done by hand, but it's easy to also write a baseline tests for this where Git itself provides the expectations.

I would only want to test the that the branch-handling is correct, and the time-computation itself.

Reply:
similar to the other test where check the first_refence name is "refs/heads/main" right? For git based testing I would like some pointers on how to do that.

Reply:
i looked at the code a bit but I understand the how the complex_graph baseline works (I am on windows so I can't just run the script locally) to see how many entries it has and do exact time test etc (especially how the dates for test entries are created). We also need a baseline with no entries and I have no idea on how to do that.

Reply:
Let's keep it with hand-written tests - these are enough to get this right I am sure. Baselines are usually for very complex behaviours that should be mimicked, but I don't think that's the case here.

Reply:
so do I just create an history using the gitoxide api. I was trying to to mimick the other test https://github.com/dvtkrlbs/gitoxide/blob/6b2643530e04bccfc741777a4b84d645d1537574/gix/tests/gix/revision/spec/from_bytes/reflog.rs#L41 but I don't don't understand the repo creation part.

Reply:
No worries - if you think this could work, and you have verified it manually with `cargo run --bin gix -- rev parse`  a little, then I think this PR can be ready for review and I take it from there.

Reply:
I will do more manual testing and mark it as ready for review once I am confident about the changes. Thank you